recipes/qt/6.x.x/conanfile.py

```diff
@@ -167,6 +167,10 @@ def _get_module_tree(self):
 
         return self._submodules_tree
 
+    def _with_system_dependency(self, dep):
```

Comment:
shouldn't this be named `_with_conan_dependency` ? in conan context, system dependencies are the one installed by the distro's package manager

Reply:
Yeah that is probably better, I just went with system because that is what the qt cmake define calls it.